src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -297,7 +315,7 @@ afterEach(() => {
   clock = clock.uninstall();
 });
 
-const setupMocks = () => {
+const setupGetNewTokensMocks = () => {
```

Comment:
This change is needed? Seems to be only a rename.

Reply:
Yes, it's just a rename to better explain what setup is doing here